lib/actions/user.js

```diff
@@ -323,7 +323,7 @@ export function createOrUpdateUser(userData, silentOnSuccess = false, intl) {
 
     // TODO: improve the UI feedback messages for this.
     if (status === 'success' && returnedUser) {
-      if (!silentOnSuccess) {
+      if (!silentOnSuccess && intl) {
```

Comment:
When we call this method from within redux we don't have access to an `intl` object and it's therefore null. Since these user updates are never based on user input anyway this is no big deal (`silentOnSuccess` is true)